design-system/packages/options/src/index.tsx

```diff
@@ -162,50 +168,52 @@ export const Options = ({ components: propComponents, ...props }: OptionsProps)
   const displaySearch = true;
```

Comment:
@mitchellhamilton  or @JedWatson , we hard code displaySearch here. 
Is the context here, "this is hard coded for now until we decide this is necessary to add as a prop for this component"? Otherwise I'd imagine we can circumvent creating context or selectProps by just moving this into the `Control` component.